src/Http/Client/FormDataPart.php

```diff
@@ -93,12 +65,12 @@ class FormDataPart implements Stringable
      * @param string $disposition The type of disposition to use, defaults to form-data.
      * @param string|null $charset The charset of the data.
      */
-    public function __construct(string $name, string $value, string $disposition = 'form-data', ?string $charset = null)
-    {
-        $this->_name = $name;
-        $this->_value = $value;
-        $this->_disposition = $disposition;
-        $this->_charset = $charset;
+    public function __construct(
+        protected string $name,
```

Comment:
Can we have inline docblocks for the props here if needed and do the popular tools / IDE read them?

Reply:
Inline docblocks are allowed, but I don't know if it's any different than having the @param tags.

Reply:
That's good enough I guess. Should only be needed when we want to specify array shapes/types for e.g.